pyhanko/cli/commands/signing/__init__.py

```diff
@@ -146,6 +152,8 @@ def addsig(
     field,
     name,
     reason,
+    contact_info,
```

Comment:
Maybe a good opportunity to batch up the `name`, `reason`, `location`,... values into a dataclass. In the CLI subpackage, breaking the API is not a big deal (except for the plugin system, but you're not touching that).

Reply:
I really like this idea. Do you mean to batch up all signature's dictionary entries related to the signer, right?

Reply:
Yep, essentially. But now that I think of it, it's actually a bit tricky to roll that in `click`, so feel free to leave that as-is. I'll refactor the internal bits myself the next time I do some other breaking API changes.